util.js

```diff
@@ -0,0 +1,18 @@
+const IsCanonicalBase64 = require('is-canonical-base64')
```

Comment:
Would it make sense to use ssb-ref here? This code looks a lot like the implementation of ssb-ref

Reply:
Yeah likely we should migrate there. I would like the general checkers like isBoxLike